### ISSUE TITLE
The quotes in configuration.toml are not consistent

### ISSUE THREAD

**Issue by cloudxxx8** [edgexfoundry/edgex-go#3663]
# 🐞 Bug Report

### Affected Services

 The issue is located in: all services

### Is this a regression?

No

### Description and Minimal Reproduction
Almost all the configuration.toml files mix single quote (') and double quote ("), including app services and device services.
We had better decide a standard and make them consistent.
For example:
https://github.com/edgexfoundry/app-service-configurable/blob/2474bfb9299069bd6f7ffa4b5411d39705da1585/res/mqtt-export/configuration.toml#L99-L119
https://github.com/edgexfoundry/device-rest-go/blob/ebfdf0cb37e6937bdbae1f20ddf4825a64340012/cmd/res/configuration.toml#L2-L9
https://github.com/edgexfoundry/edgex-go/blob/70a7de6140c9210a4916c44f40b1630081c5c4da/cmd/core-data/res/configuration.toml#L1-L9

**EdgeX Version:**
main

**Comment by FelixTing** [comment]
FYR
The TOML spec(https://toml.io/en/v1.0.0#string) states:
- Basic strings are surrounded by quotation marks.
```
str1 = "I'm a string."
str2 = "You can \"quote\" me."
str3 = "Name\tJos\u00E9\nLoc\tSF."
```
- Literal strings are surrounded by single quotes.
```
# What you see is what you get.
path = 'C:\Users\nodejs\templates'
path2 = '\\User\admin$\system32'
quoted = 'Tom "Dubs" Preston-Werner'
regex = '<\i\c*\s*>'
```

**Comment by lenny-intel** [comment]
@FelixTing , Thank you for the spec reference! IMO, this is what we should be following.

**Comment by cloudxxx8** [comment]
fixed by #3666
